### DISCUSSION TITLE
Supabase Logs Issues

### DISCUSSION THREAD

**Discussion by ctrl-plus-w** [supabase/supabase#18504] [Questions]
When ever I try to access my app's logs I get this error :
{
  "code": 503,
  "message": "upstream connect error or disconnect/reset before headers. retried and the latest reset reason: connection timeout",
  "requestId": "92f70424-e260-44e6-82af-06b1a3eeaec0"
}
I also sometimes get an error saying "no healthy upstream".

Is this error a bug ? And if not, how can I fix it ?

**Comment by GaryAustin1** [comment]
Yes it is on the status page.
https://status.supabase.com/
![image](https://github.com/supabase/supabase/assets/54564956/c8dcb272-5d2d-415b-97f4-46007048c243)
